timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AutobatchingPaxosAcceptorNetworkClientFactory.java

```diff
@@ -28,6 +28,7 @@
 import com.google.common.util.concurrent.ListenableFuture;
```

Comment:
nit: there are unused imports in some other places as well where ExecutorService is replaced by CheckedRejectionExecutorService